.github/scripts/pr-review.py

```
@@ -0,0 +1,227 @@
+        "model": model,
+        "max_tokens": MAX_OUTPUT_TOKENS,
+        "system": system,
+        "messages": [{"role": "user", "content": user}],
+    }).encode("utf-8")
+
+    for attempt, delay in enumerate([0] + RETRY_DELAYS):
+        if delay:
+            print(f"  Retrying in {delay}s (attempt {attempt + 1})...", file=sys.stderr)
+            time.sleep(delay)
+        try:
+            req = urllib.request.Request(API_URL, data=payload, headers=headers, method="POST")
```

Ruff (0.15.2)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)